packages/components/Button/src/Button.settings.ts

```diff
@@ -29,13 +29,12 @@ export const settings: IComposeSettings<IButtonType> = [
     stack: {
       style: {
         display: 'flex',
-        paddingStart: 16,
-        paddingEnd: 16,
+        paddingStart: 10,
+        paddingEnd: 10,
         alignItems: 'center',
         flexDirection: 'row',
         alignSelf: 'flex-start',
         minHeight: 32,
-        minWidth: 80,
         justifyContent: 'center',
       },
```

Comment:
How does removing minWidth affect the style of a button? If someone created a button without any content (or icon), would the button still be visible? And what's the reason for changing the padding from 16 to 10?

Reply:
Ah minWidth should be added back actually. I meant to change it to 32. An IconButton and Button without content should be a square 32x32 button by default. The padding change was also for this reason.

Reply:
I see, thanks!